src/interface/src/app/plan/plan-summary/saved-scenarios/saved-scenarios.component.html

```diff
@@ -79,23 +64,27 @@
 
         <mat-header-row *matHeaderRowDef="displayedColumns"></mat-header-row>
         <mat-row
-          class="scenario-row"
+          [ngClass]="{
+            'scenario-row': highlightedId != row.id,
+            highlight: highlightedId == row.id
+          }"
           *matRowDef="let row; columns: displayedColumns"
-          (click)="viewScenario(row.id)"></mat-row>
+          (click)="highlightScenario(row.id)"
+          (dblclick)="viewScenario()"></mat-row>
       </mat-table>
     </mat-card-content>
 
     <mat-card-actions align="end">
       <button
         mat-button
-        [disabled]="!showDeleteButton()"
+        [disabled]="highlightedId == '-1'"
```

Comment:
Should use triple equal here, also, shouldn't -1 be a number?
If not maybe we should use `null` instead when you don't have any selected

Reply:
Changed it to null!